clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVMIR.cpp

```diff
@@ -267,4 +267,4 @@ void registerCIRDialectTranslation(mlir::MLIRContext &context) {
   context.appendDialectRegistry(registry);
 }
 } // namespace direct
-} // namespace cir
+} // namespace cir
```

Comment:
Seems like the only change in this file is to remove the newline at the end of it? I think it's more conventional to keep newlines at the end of files in general, but we should also undo this particular change (since there's no other changes to the file).

Reply:
got rid of the change.